packages/google-closure-compiler/docs/grunt.md

```diff
@@ -28,6 +28,7 @@ grunt.initConfig({
         'dest/output.min.js': ['src/js/**/*.js']
       },
       options: {
+        compile_in_batches: require('os').cpus().length,
```

Comment:
This isn't a closure-compiler option. I've tried to reserve the `options` object for only direct options of the compiler proper. Plugin specific options need pulled out in a similar to how the `platform` argument works. See https://github.com/google/closure-compiler-npm/blob/master/packages/google-closure-compiler/docs/grunt.md#specifying-extra-java-arguments for an example.

Reply:
Mmm! Just wrong example. It is already parameter like "platforms". Will fix example